data/rule_data.yml

```diff
@@ -20,3 +20,6 @@ rule_data:
   allowed_java_component_sources:
   - redhat
   - rebuilt
+
+  # See also the additional default rule data values defined in
+  # https://github.com/hacbs-contract/ec-policies/blob/main/policy/lib/rule_data.rego#L6
```

Comment:
We probably will never change this, but if we do add/remove lines to that file, it may break this link. Maybe just omit the line number altogether?